var/www/html/tools/opcache/index.php

```diff
@@ -0,0 +1,744 @@
+<?php
```

Comment:
I dont' actually know the original source of this file. it's something we have sitting on several client servers to check on the status of the opcache. I had intended it to be removed after I was done ensuring that part was working correctly.